performancetest/src/main/java/org/corfudb/performancetest/CorfuTablePerformanceTest.java

```diff
@@ -0,0 +1,193 @@
+package org.corfudb.performancetest;
+
+import com.google.common.reflect.TypeToken;
+import lombok.extern.slf4j.Slf4j;
+import org.corfudb.runtime.collections.CorfuTable;
+import org.junit.Test;
+import java.io.IOException;
+import java.util.*;
+
+/**
+ * CorfuTablePerformanceTest is the subclass of PerformanceTest.
+ * This class is used for CorfuTable performance test, mainly for put and get operations.
+ * @author Lin Dong
+ */
+@Slf4j
+public class CorfuTablePerformanceTest extends PerformanceTest{
+    /**
+     * Seed used by Random Object.
+     */
+    private final long seed;
+    /**
+     * Boundary of the integer generated randomly.
+     */
+    private final int randomBoundary;
+    /**
+     * The time each CorfuTable performance test runs.
+     */
+    private final long time;
+    /**
+     * The max number of keys can generated by KeyValueManager.
+     */
+    private final int keyNum;
+    /**
+     * The size of value generated by KeyValueManager.
+     */
+    private final int valueSize;
+    /**
+     * Random Object.
+     */
+    private final Random random;
+    /**
+     * KeyValueManager for generating and managing key value pairs.
+     */
+    private final KeyValueManager keyValueManager;
+
+    /**
+     * Constructor. Initiate related variables.
+     */
+    public CorfuTablePerformanceTest() {
+        seed = Long.parseLong(PROPERTIES.getProperty("tableSeed", "1024"));
+        randomBoundary = Integer.parseInt(PROPERTIES.getProperty("tableRandomBoundary","10"));
+        time = Long.parseLong(PROPERTIES.getProperty("tableTime", "100"));
+        keyNum = Integer.parseInt(PROPERTIES.getProperty("keyNum", "10"));
+        valueSize = Integer.parseInt(PROPERTIES.getProperty("valueSize", "1024"));
+        random = new Random(seed);
+        keyValueManager = new org.corfudb.performancetest.KeyValueManager(keyNum, valueSize);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'org.corfudb.performancetest.KeyValueManager' due to existing same package import 'org.corfudb.performancetest.*'](https://app.codacy.com/app/annymartinez.m/CorfuDB/pullRequest?prid=4651454)